torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -56,10 +58,13 @@ def from_plain(
     ):
         pass
 
+    @torch._dynamo.disable
     def __repr__(self):
-        int_data, scale, zero_point = self.get_plain()
-        layout_type = self.get_layout_type()
-        return f"{self.__class__.__name__}(int_data={int_data}, scale={scale}, zero_point={zero_point}, layout_type={layout_type})"
+        # This is a hack, torch.compile tries to trace the __repr__ function which then calls `dequantize` function, causing an error.
```

Comment:
can these changes be reverted now with dynamo.disable decorator?

Reply:
yeah, will take care of that